README.md

```diff
@@ -25,63 +36,53 @@ The I2Cwrapper framework consists of **four basic components.** The first two dr
 
 ### Limitations for module authors
 
    - No **serialization protocol** is used at the moment, so the implementation is machine dependent in regard to the endians and sizes of data types. Modules will have to take care that transmitted commands and requests will transmit defined amounts of bytes by using typeguards for ambiguously sized datatypes like *int*.
    - Modules use one byte **command codes** for each distinct function call to the target. At the moment, no mechanism is in place to prevent newly developed modules from reusing codes already used by another module or by one of the I2Cwrapper core functions. Note that this will only lead to problems if two modules are used concurrently by a target device. The [I2Cwrapper documentation](https://ftjuh.github.io/I2Cwrapper/docs/class_i2_cwrapper.html) has a list of code ranges used by the currently available modules. Strictly reserved ranges are 0-9 and 240-255.
 
-See [below](#How to add new modules) if you are interested in writing a new module and implementing your own target device.
+See [below](#how-to-add-new-modules) if you are interested in writing a new module and implementing your own target device.
 
 # Usage
 
 ## Installation
 
 ~~Install I2Cwrapper from the Arduino library manager (not available yet).~~ Until I2Cwrapper is picked up by the Arduino library manager, you'll have to [install it manually](https://docs.arduino.cc/software/ide-v1/tutorials/installing-libraries#manual-installation) by copying the I2Cwrapper folder to your Arduino library folder and (re)starting the Arduino software.  You'll find an I2Cwrapper examples folder after successful installation.
 
-## Configuring and uploading the firmware
+## Target side: configuring and uploading the firmware
 
 * **Open [firmware.ino](https://github.com/ftjuh/AccelStepperI2C/blob/controller/firmware/firmware.ino)**  from the examples menu of the Arduino editor, you'll find it in the the I2Cwrapper submenu. It will open multiple tabs, among them one for each available module in the firmware subfolder.
-* Go to  the `firmware_modules.h` tab and **select the modules** you want by (un)commenting them. For a first test, start with the PinI2C module, it is the simplest and doesn't need any extra hardware.
+* Go to  the [`firmware_modules.h`](firmware/firmware_modules.h) tab and **select the modules** you want by (un)commenting them. For a first test, start with the `PinI2C` module, it is the simplest and doesn't need any extra hardware.
```

Comment:
PinI2C is a label here, not code, let's not overdo.

Reply:
code markup removed.